core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -25,51 +25,74 @@
-      </collection-details-editor-directive>
-      <a ng-href="/collection/<[collection.getId()]>" target="_blank">Go to Collection</a>
-
-      <!-- List of acquired and pre-requisite skills -->
-      <br>
-      <br>
-      <strong>Skills</strong>
-      <collection-skill-list-directive collection-skill-list="collectionSkillList">
-      </collection-skill-list-directive>
-
-      <!-- List of explorations -->
-      <br>
-      <strong>Explorations</strong>
-      <ul>
-        <li ng-repeat="node in collection.getBindableCollectionNodes()">
-          <collection-node-editor-directive collection="collection" collection-node="node" update-skill-list="updateSkillList()">
-          </collection-node-editor-directive>
-        </li>
-      </ul>
-
-      <!-- Collection changes to be commited -->
-      <h3><span ng-if="getChangeListCount() > 0"> <em>(<[getChangeListCount()]> changes pending)</em></h3>
-      <br>
-      <input ng-model="commitMessage" type="text" placeholder="Commit Message"><button ng-click="saveCollection(commitMessage)">Save Collection</button>
-      <button ng-click="publishCollection()" ng-if="!isPublic">Publish Collection</button>
+    <div ng-if="getTabStatuses().active === 'main'">
+      {% include 'collection_editor/editor_tab/collection_main.html' %}
+    </div>
+
+    <div ng-if="getTabStatuses().active === 'preview'">
+      {% include 'collection_editor/preview_tab/collection_preview.html' %}
+    </div>
+
+    <div ng-show="getTabStatuses().active === 'settings'">
+      {% include 'collection_editor/settings_tab/collection_settings.html' %}
+    </div>
+
+    <div ng-show="getTabStatuses().active === 'history'">
+      {% include 'collection_editor/history_tab/collection_history.html' %}
+    </div>
+
+    <div ng-show="getTabStatuses().active === 'feedback'">
+      {% include 'collection_editor/feedback_tab/collection_feedback.html' %}
+    </div>
+
+    <div ng-show="getTabStatuses().active === 'stats'">
+      {% include 'collection_editor/statistics_tab/collection_statistics.html' %}
     </div>
   </div>
 
-  {% include 'collection_editor/collection_details_editor_directive.html' %}
-  {% include 'collection_editor/collection_node_editor_directive.html' %}
-  {% include 'collection_editor/collection_skill_list_directive.html' %}
+  {% include 'collection_editor/editor_tab/collection_details_editor_directive.html' %}
+  {% include 'collection_editor/navbar/collection_editor_navbar_breadcrumb_directive.html' %}
+  {% include 'collection_editor/navbar/collection_editor_navbar_directive.html' %}
+  {% include 'collection_editor/editor_tab/collection_node_editor_directive.html' %}
+  {% include 'collection_editor/editor_tab/collection_skill_list_directive.html' %}
 {% endblock content %}
 
 {% block footer_js %}
   {{ super() }}
   <script>
+    {{ include_js_file(
+      'collection_editor/editor_tab/CollectionDetailsEditorDirective.js') }}
```

Comment:
Please keep all of these on one line (ditto below). We don't have any column count limits for HTML files since it results in things being a bit unreadable.